packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -235,18 +262,22 @@ export class SerializedStateManager {
 				specifiedVersion,
 			);
 			const baseSnapshotTree: ISnapshotTree | undefined = getSnapshotTree(baseSnapshot);
+			const attributes = await getDocumentAttributes(this.storageAdapter, baseSnapshotTree);
 			// non-interactive clients will not have any pending state we want to save
 			if (this.offlineLoadEnabled) {
-				const snapshotBlobs = await getBlobContentsFromTree(baseSnapshot, this.storageAdapter);
```

Comment:
Previously an error here would cause Container.load to error out (presumably to be caught by the caller), and no IContainer would be given back to the customer.

I think the new flow here means that the Container.load finishes before the error would occur - what should happen to the IContainer that the customer is holding in the case of an error after-the-fact?  Close, event, etc.?

Reply:
i don't want load to fail. i'd rather have getpendingstate fail, which will happen now. the lower layers should do a bunch of retries, and we have logging to figure out problems and interate

Reply:
this code never runs today as it is under the offlineLoadEnabled flag. i want to turn that flag on, but keep the calling pattern the as today without the flag